.github/workflows/test.yml

```diff
@@ -13,47 +13,94 @@ on:
     inputs:
       tag:
         required: true
+        description: 'RocketChat tag to test'
       chartVersion:
+        description: 'Chart version to test'
         required: true
+  push:
+    paths:
+      - 'rocketchat/**'
+      - 'bats/**'
+      - 'mock/**'
+      - '.github/workflows/test.yml'
+      - 'task.bash'
 
 env: 
   REPO: 'RocketChat/helm-charts'
   TESTS_REPO: 'RocketChat/public-releases'
 
-
 jobs:
-  can-i-run:
+  # this 3 jobs will run in parallel for warming up the workflow
```

Comment:
why? caching the action images? not sure if cache hit will occur given all jobs run in parallel, even the one that uses the actions